VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,71 @@
+
+To supervise the initiative, it was decided to create a Technical Steering Committee (TSC), made up of the people who contribute most actively to the project. Anyone with interest can become a member of the TSC. All you have to do is become a maintainer of one of the AsyncAPI projects. To become a maintainer, it is necessary to regularly contribute to one of the projects; after that, other maintainers will invite them to join. 
+
+The TSC is constituted by maintainers because the goal is to empower "working" individuals, not companies that "pay". In other words, it provides equal power to individual and corporate contributors.
+
+The TSC's system of operation is by voting. An issue or debate is posted, and the members vote asynchronously and publicly on their decision. The value of the vote of all contributors is equal, thus ensuring the project that there is no risk of one company having a majority of the voters. That is why there is a rule that only 1/4 of the voters can be affiliated with the same company.
+
+## Putting theory into practice
+
+Summing up, AsyncAPI is committed to democratic management based on two principles: horizontality in decision making and transparency in project management. From the beginning of the specification, AsyncAPI has relied on collective effort and group identity. The means are as crucial as the goal. Responsibilities are shared, understanding that a community can only achieve success jointly.
+
+For doing so, AsyncAPI has several tools that ensure transparency within the project’s open-governance management. Let's see how theory is put into practice:
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
+
+The TSC makes decisions on financial expenses through voting.
+
+- **Communication**: To ensure that the principle of transparency governs communication, everything is said and done in public, no matter how insignificant or essential it may be. The community uses GitHub as the first communication and work platform. All decisions are taken there. Secondly, Slack is used as a means of sync communication enablement and keeping the closer bond with the community on daily basis.
+
+- **Meetings**: There are different meeting points where the community can meet, propose or discuss any topic related to AsyncAPI. Any interested person can join these meetings and raise any thoughts.
+
+  Even though there are synchronous meetings, there is a written rule that no decision making is done during them. After something is discussed on a        meeting, the community needs to have time to react, watch the recording and decide. Is then when decisions can be finally made, asynchronously.
```

Comment:
What is a synchronous and asynchronous meeting?

**meetings are recorded so that people who did not attend can watch the discussion later and participate in the decision-making process**